main.go

```diff
@@ -362,6 +377,7 @@ func getFromMetadata(urlStr string) ([]byte, error) {
 
 type config struct {
 	XdsServers                         []server                             `json:"xds_servers,omitempty"`
+	Authorities                        map[string]authority                 `json:"authorities,omitempty"`
```

Comment:
This won't compile because you have gotten rid of the `authority` struct now. Thinking about this more, I think it is better if we have the struct, but the struct can be an empty one, with a TODO saying, we will add fields to it when we make them customizable.

Because without the struct, this would have to be:
```golang
Authorities map[string]interface{}
```
Better to avoid the `interface{}` when we can.

Reply:
got it. updating this